media/js/firefox/new/all-test-variations.js

```diff
@@ -0,0 +1,133 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, dataLayer) {
+    'use strict';
+
+    // check page version for tests
+    var version = Number($('#masthead').data('version'));
+
+    // adds sort value & (optional) new text to direct download li
+    var prepLi = function(li) {
+        var $li = $(li);
+        var $liA = $li.find('a');
+        var newOsName;
+
+        switch ($liA.text()) {
+        case 'Windows':
```

Comment:
This needn't block this PR, but just to note we're adding another download button type in #4413 for sha-1 downloads. I'll likely need to update the logic for what's displayed in this test after this merges to master.